index.html

```diff
@@ -9,10 +9,8 @@
 
     <!-- Bootstrap -->
     <link href="css/bootstrap.min.css" rel="stylesheet">
-
-    <!-- Custom -->
-    <link href="css/custom.css" rel="stylesheet">
-
+    <link href="css/bootstrap-theme.min.css" rel="stylesheet">
```

Comment:
This isn't the place to add your own code. Create your own index.html file under your student folder.